package.json

```diff
@@ -76,7 +76,6 @@
     "babel-plugin-dynamic-import-node": "^2.1.0",
     "babel-plugin-emotion": "^10.0.23",
     "bufferutil": "^4.0.1",
-    "caniuse-lite": "^1.0.30001099",
```

Comment:
Hmm where does this change come from? Mention in PR description

Reply:
Hm... I only did a yarn install and got this 🤔 Should I remove the changes here as well as the ones on yarn.lock? Because I honestly don't know what they will do 😅

Reply:
hmm yeah I wonder why that is -- yarn install doesn't change package.json as far as I know